src/foraging_gui/MyFunctions.py

```diff
@@ -813,7 +814,8 @@ def _ShowInformation(self):
                         'Current right block: ' + str(self.BS_CurrentBlockTrialNV[1]) + '/' +  str(self.BS_CurrentBlockLenV[1])+'\n\n'
                         'Responded trial: ' + str(self.BS_FinisheTrialN) + '/'+str(self.BS_AllTrialN)+' ('+str(np.round(self.BS_RespondedRate,2))+')'+'\n'
                         'Reward Trial: ' + str(self.BS_RewardTrialN) + '/' + str(self.BS_AllTrialN) + ' ('+str(np.round(self.BS_OverallRewardRate,2))+')' +'\n'
-                        'Total Reward (ul;excluding manual rewards): '+ str(self.BS_RewardN)+' : '+str(np.round(self.BS_TotalReward,3)) +'\n'
+                        'Earned Reward (ul): '+str(np.round(self.win.Earnedwater*1000,2)) +'\n'
+                        'Earned Reward (ul;excluding manual rewards): '+ str(self.BS_RewardN)+' : '+str(np.round(self.BS_TotalReward,3)) +'\n'
```

Comment:
So maybe here instead of "Earned Reward" and "Earned, excluding manual" we should report "Earned Reward" and "Water in session"

Reply:
I agree. Already changed.